cli/deploy-managed-online-endpoint-access-resource-sai.sh

```diff
@@ -39,44 +39,36 @@ az storage container create --account-name $STORAGE_ACCOUNT_NAME --name $STORAGE
 az storage blob upload --account-name $STORAGE_ACCOUNT_NAME --container-name $STORAGE_CONTAINER_NAME --name $FILE_NAME --file endpoints/online/managed/managed-identities/hello.txt
 # </upload_file_to_storage>
 
+cat endpoints/online/managed/managed-identities/1-sai-create-endpoint.yml
+
 # <create_endpoint>
-az ml endpoint create --name $ENDPOINT_NAME -f endpoints/online/managed/managed-identities/1-sai-create-endpoint.yml
+az ml online-endpoint create --name $ENDPOINT_NAME -f endpoints/online/managed/managed-identities/1-sai-create-endpoint.yml
 # </create_endpoint>
 
 # <check_endpoint_Status>
-az ml endpoint show --name $ENDPOINT_NAME
+az ml online-endpoint show --name $ENDPOINT_NAME
 # </check_endpoint_Status>
 
-endpoint_status=`az ml endpoint show --name $ENDPOINT_NAME --query "provisioning_state" -o tsv`
-echo $endpoint_status
-if [[ $endpoint_status == "Succeeded" ]]
-then
-  echo "Endpoint created successfully"
-else 
-  echo "Endpoint creation failed"
-  exit 1
-fi
-
 # role assignment fails without sleep statement
 sleep 60
 
 # <get_system_identity>
-system_identity=`az ml endpoint show --name $ENDPOINT_NAME --query "identity.principal_id" -o tsv`
+system_identity=`az ml online-endpoint show --name $ENDPOINT_NAME --query "identity.principal_id" -o tsv`
 # </get_system_identity>
 
 # <give_permission_to_user_storage_account>
-az role assignment create --assignee $system_identity --role "Storage Blob Data Reader" --scope $storage_id
+az role assignment create --assignee-object-id $system_identity --assignee-principal-type ServicePrincipal --role "Storage Blob Data Reader" --scope $storage_id
 # </give_permission_to_user_storage_account>
 
 # <deploy>
-az ml endpoint update --name $ENDPOINT_NAME --deployment blue --file endpoints/online/managed/managed-identities/2-sai-deployment.yml --set deployments[0].environment_variables.STORAGE_ACCOUNT_NAME=$STORAGE_ACCOUNT_NAME deployments[0].environment_variables.STORAGE_CONTAINER_NAME=$STORAGE_CONTAINER_NAME deployments[0].environment_variables.FILE_NAME=$FILE_NAME
+az ml online-deployment create --endpoint-name $ENDPOINT_NAME --name blue --file endpoints/online/managed/managed-identities/2-sai-deployment.yml --set environment_variables.STORAGE_ACCOUNT_NAME=$STORAGE_ACCOUNT_NAME environment_variables.STORAGE_CONTAINER_NAME=$STORAGE_CONTAINER_NAME environment_variables.FILE_NAME=$FILE_NAME
```

Comment:
need to add --all-traffic. We can remove the separate command for traffic update  at line 68